std/datetime/package.d

```diff
@@ -1,98 +1,76 @@
-    be done on a series of time points.
-
-    The types that the typical user is most likely to be interested in are
-    $(REF Date,std,_datetime,date) (if they want dates but don't care about
-    time), $(REF DateTime,std,_datetime,date) (if they want dates and times
-    but don't care about time zones), $(REF SysTime,std,_datetime,systime) (if
-    they want the date and time from the OS and/or do care about time zones),
-    and StopWatch (a platform-independent, high precision stop watch).
-    $(REF Date,std,_datetime,date) and $(REF DateTime,std,_datetime,date) are
-    optimized for calendar-based operations, while
-    $(REF SysTime,std,_datetime,systime) is designed for dealing with time from
-    the OS. Check out their specific documentation for more details.
-
-    To get the current time, use $(REF Clock.currTime,std,_datetime,systime).
-    It will return the current time as a $(REF SysTime,std,_datetime,systime). To
-    print it, `toString` is sufficient, but if using `toISOString`,
-    `toISOExtString`, or `toSimpleString`, use the corresponding
-    `fromISOString`, `fromISOExtString`, or `fromSimpleString` to
-    create a $(REF SysTime,std,_datetime,systime) from the string.
-
---------------------
-auto currentTime = Clock.currTime();
-auto timeString = currentTime.toISOExtString();
-auto restoredTime = SysTime.fromISOExtString(timeString);
---------------------
-
-    Various functions take a string (or strings) to represent a unit of time
-    (e.g. $(D convert!("days", "hours")(numDays))). The valid strings to use
-    with such functions are `"years"`, `"months"`, `"weeks"`,
-    `"days"`, `"hours"`, `"minutes"`, `"seconds"`,
-    `"msecs"` (milliseconds), `"usecs"` (microseconds),
-    `"hnsecs"` (hecto-nanoseconds - i.e. 100 ns), or some subset thereof.
-    There are a few functions in core.time which take `"nsecs"`, but because
-    nothing in std.datetime has precision greater than hnsecs, and very little
-    in core.time does, no functions in std.datetime accept `"nsecs"`.
-    To remember which units are abbreviated and which aren't,
-    all units seconds and greater use their full names, and all
-    sub-second units are abbreviated (since they'd be rather long if they
-    weren't).
-
-    Note:
-        $(REF DateTimeException,std,_datetime,date) is an alias for
-        $(REF TimeException, core,time), so you don't need to worry about
-        core.time functions and std.datetime functions throwing different
-        exception types (except in the rare case that they throw something other
-        than $(REF TimeException, core,time) or
-        $(REF DateTimeException,std,_datetime,date)).
+    $(UL
+        $(LI $(MREF std, _datetime, date))
+        $(LI $(MREF std, _datetime, timezone))
+        $(LI $(MREF std, _datetime, systime))
+        $(LI $(MREF std, _datetime, interval))
+        $(LI $(MREF std, _datetime, stopwatch))
```

Comment:
Maybe add a short, one-line description of each of these packages, s.t. the user knows what the difference betwen date and systime is?